source/fuzz/CMakeLists.txt

```diff
@@ -311,6 +313,7 @@ if(SPIRV_BUILD_FUZZER)
         fuzzer_pass_split_blocks.cpp
         fuzzer_pass_swap_commutable_operands.cpp
         fuzzer_pass_swap_conditional_branch_operands.cpp
+        fuzzer_pass_swap_functions.cpp
```

Comment:
I get a compile error (warning) because the transformation is not handled in transformation.cpp:

```
/data/git/SPIRV-Tools/source/fuzz/transformation.cpp:114:11: error: enumeration value 'kSwapTwoFunctions' not handled in switch [-Werror,-Wswitch]
  switch (message.transformation_case()) {

```

Reply:
Added `kSwapTwoFunctions` to `transformation.cpp` and is now fixed.